bootstrap/bootstrap-pod.yaml

```diff
@@ -37,6 +37,7 @@ spec:
           fieldRef:
             fieldPath: spec.nodeName
   hostNetwork: true
+  terminationGracePeriodSeconds: 130
```

Comment:
comment about how you chose this value please.  Even if you just pulled one out.

Reply:
> comment about how you chose this value please...

```console
$ git show --stat ada3fe20ad6 | grep -A2 'two-minute\|terminationGracePeriodSeconds'
    removing that code here in favor of the two-minute timeout from
    b30aa0ecdb (pkg/cvo/metrics: Graceful server shutdown, 2020-04-15, #349).
    We still exit immediately on a second TERM, for folks who get
--
    Set terminationGracePeriodSeconds 130 to extend the default 30s [3],
    to give the container the full two-minute graceful timeout window
    before the kubelet steps in with a KILL.
```